server/xpub/schema/migrations/1536842027-opposed-reviewers.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE manuscript
+  ADD COLUMN opposed_reviewers_reason TEXT,
+  DROP COLUMN opposed_reviewers;
```

Comment:
Same as above, why is this being removed & the same time as we're adding a reason into the manuscript? :confused: